sklearn/tests/test_multioutput.py

```diff
@@ -85,3 +89,91 @@ def test_multi_target_sample_weights():
 
     X_test = [[1.5,2.5,3.5], [3.5,4.5,5.5]]
     assert_almost_equal(rgr.predict(X_test), rgr_w.predict(X_test))
+
+# Import the data
+iris = datasets.load_iris()
+# create a multiple targets by randomized shuffling and concatenating y.
+X = iris.data
+y1 = iris.target
+y2 = shuffle(y1, random_state=1)
+y3 = shuffle(y1, random_state=2)
+y = np.column_stack((y1, y2, y3))
+n_samples, n_features = X.shape
+n_outputs = y.shape[1]
+n_classes = len(np.unique(y1))
+
+
+def test_multi_output_classification():
+    # test if multi_target initializes correctly with base estimator and fit
+    # assert predictions work as expected for predict, prodict_proba and score
+
+    forest = RandomForestClassifier(n_estimators=10, random_state=1)
+    multi_target_forest = MultiOutputClassifier(forest)
+    # assert_equal(forest, multi_target_forest.estimator)
+
+    # train the multi_target_forest and also get the predictions.
+    multi_target_forest.fit(X, y)
+
+    predictions = multi_target_forest.predict(X)
+    assert_equal((n_samples, n_outputs), predictions.shape)
+
+    predict_proba = multi_target_forest.predict_proba(X)
+    assert_equal((n_samples, n_classes, n_outputs), predict_proba.shape)
+
+    assert_array_equal(np.argmax(predict_proba, axis=1), predictions)
+
+    # train the forest with each column and assert that predictions are equal
+    for i in range(3):
+        forest_ = clone(forest)  # create a clone with the same state
+        forest_.fit(X, y[:, i])
+        assert_equal(list(forest_.predict(X)), list(predictions[:, i]))
+        assert_almost_equal(list(forest_.predict_proba(X)),
+                            list(predict_proba[:, :, i]), decimal=1)
+
+
+def test_multiclass_multioutput_estimator():
```

Comment:
No, I don't think so. Since ``MultiOutputClassifier(OneVsRestClassifier(.. ))`` works with fitting the OneVsRest estimator as base estimator for each multiclass single output and thus fits for n_outputs, but then using ``OneVsRestClassifier(MultiOutputClassifier(..))`` is not possible since OneVsRest expects the base estimator to be a single output estimator but MultiOutputEstimator has more than one outputs. I do remember trying this once, and this is what I could discern from the error. Please do let me know if you feel that this functionality can be possible.